### ISSUE TITLE
Question Regarding pitch_dict.pt File

### ISSUE THREAD

**Issue by hayderkharrufa** [nipponjo/tts-arabic-pytorch#12]
Hello.

There's a file named pitch_dict.pt used in FastPitch training. What is this file and where can I get it?

I'm trying to train with FastPitch to check if the issue in [#10](https://github.com/nipponjo/tts-arabic-pytorch/issues/10) is also present here or it's just for Tacotron.

Thank you.

**Comment by nipponjo** [comment]
Hello,

the FastPitch model conditions the output (mel spectrogram) on the pitch values of the speaker. To train the pitch predictor, the pitch values of the training set are needed, which can be extracted using the `extract_f0.py` script. The `wavs_path` inside the script has to be adapted.

In the last push, I set the learning rate for the Tacotron2 back to 1e-3, which should be more stable. Still I don't know if training from scratch actually works, as I used pretrained weights (trained in LJSpeech) for both models.

**Comment by hayderkharrufa** [comment]
Ah, I didn't notice the extract_f0.py script as I have an older version of the repo. I'm trying to train from scratch as I've rerecorded the Arabic speech corpus with my voice and want to ensure that the recordings are of good quality. I tried training Tacotron 2 from scratch with your latest push, still didn't get the results I were getting from the 1003703bb21af7c109b2926d37ca287dfa756a5c version.

Thank you for your assistance. I will conclude this issue now.
